server/src/config.ts

```diff
@@ -1,10 +1,29 @@
-/* tslint:disable:max-line-length */
+import { DbConfig } from "./models/index";
+
 export class Config {
   public static DB_USER: string = "postgres";
   public static DB_PW: string = "socobo";
-  public static DB_URL_TEST: string = `postgres://${Config.DB_USER}:${Config.DB_PW}@localhost:5432/socobo_test?ssl=false`;
-  public static DB_URL_DEV: string = `postgres://${Config.DB_USER}:${Config.DB_PW}@localhost:5432/socobo_dev?ssl=false`;
-  public static DB_URL: string = `postgres://${Config.DB_USER}:${Config.DB_PW}@localhost:5432/socobo?ssl=false`;
+  public static DB_CONFIG_TEST: DbConfig = {
```

Comment:
Ah ok, now I understand your point and how you planned this! Ok in this case I would think we either need to stick with the url strings or we could also provide each value of the db config as a single env variable. Another approach would be to not use environment variables. Instead we could use a separate config file for the production environment which would not be committed to github.

So I think the url approach is not the most beautiful but the simples one and I would stick with this. What do you think? But the config file approach would be more flexible since we could change values at runtime without redeployment. What do you think?